### PR TITLE
Models to merge part3

### PR DESCRIPTION
**Pull request by alexandrabal**

### CODE REVIEW

File: final_project/models.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+'''class Meta- nested class, tells Django this is not a real class and to not bring it in as a table,
+therefore it's an abstract class'''
```
Comment (mihaitzzza) [inline final_project/models.py]: `abstract = True` property is the reason why this class/model is abstract, not the `Meta` class.

File: products/apps.py [hunk 2]
```diff
@@ -2,4 +2,4 @@
 
 
 class StoresConfig(AppConfig):
```
Comment (mihaitzzza) [inline products/apps.py]: You should also rename the name of the class and the directory.

File: users/models.py [hunk 3]
```diff
@@ -1,23 +1,26 @@
 from django.db import models
 from django.contrib.auth import get_user_model
 from utils.constants import BILLING_ADDRESS, SHIPPING_ADDRESS
+from final_project.models import CustomModel
 
 AuthUserModel = get_user_model()
-
+"This represents the usermodels, user model default from django added in a variable"
```
Comment (mihaitzzza) [inline users/models.py]: This is still a string data ... why are you throwing a string data randomly in your code?

**Review by mihaitzzza — CHANGES_REQUESTED** [review]

File: users/models.py (hunk 3 above)
Comment (alexandrabal) [reply users/models.py]: I meant to put ''' sorry :D

**Pull request merged**